keps/prod-readiness/sig-node/2535.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 2535
 alpha:
   approver: "@jpbetz"
+beta:
+  approver: "@jpbetz"
```

Comment:
@jpbetz :wave: 
I chose you because you were doing PRR for alpha, please let me know in case you would like me to pick somebody else :pray: